File: javascript/uri-online-judge/1008.js
const fs = require('fs').readFileSync('/dev/stdin', 'utf-8');
const lines = fs.split(/\n/);

const info = lines.map( item => Number( item ));

const number      = info.shift();
const workedHours = info.shift();
const hourlyWage  = info.shift();

const salary = workedHours * hourlyWage;

console.log(`NUMBER = ${ number }`);
console.log(`SALARY = U$ ${ salary.toFixed( 2 ) }`);

File: java/uri-online-judge/1008.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Main {
  public static void main(String []args) throws IOException {
    BufferedReader stdin = new BufferedReader( new InputStreamReader( System.in ));

    int id = Integer.parseInt( stdin.readLine() );
    int workedHours = Integer.parseInt( stdin.readLine() );
    double hourlyPrice = Double.parseDouble( stdin.readLine() );

    double salary = workedHours * hourlyPrice;

    System.out.printf("NUMBER = %d\n", id);
    System.out.printf("SALARY = U$ %.2f\n", salary);
  }
}
